src/schema/resolvers/signup_resolver.js

```diff
@@ -11,10 +11,12 @@ const resolvers = {
       if (count > 0) {
         return { authError: 'The username already exists' }
       } else {
-        const user = models.user.create({ username, password: password, lastName: lastName, firstName: firstName })
+        const newuser = await models.user.create({ username, password: password, lastName: lastName, firstName: firstName })
+        const user = { username: newuser.username }
```

Comment:
You could improve this by having a separate function that takes user's non-sensible information and returns it (username, names, etc).

Reply:
I'm nor sure if what I did is what you asked for, but I believe I somehow managed to make it simpler and  started the object with the shorthand property.